flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml

```diff
@@ -142,18 +149,19 @@ under the License.
 								<filter>
 									<artifact>org.apache.hive:hive-exec</artifact>
 									<excludes>
-										<exclude>org/apache/hadoop/hive/conf/HiveConf.class</exclude>
-										<exclude>org/apache/hadoop/hive/metastore/HiveMetaStoreClient.class</exclude>
+										<exclude>avro/shaded/**</exclude>
 										<exclude>org/apache/avro/**</exclude>
 										<exclude>META-INF/maven/org.apache.avro/**</exclude>
+										<exclude>org/apache/hadoop/hive/conf/HiveConf*</exclude>
```

Comment:
Adding these changes removes all the warning in the build. 

```
[INFO] Excluding org.apache.flink:flink-shaded-force-shading:jar:15.0 from the shaded jar.
[INFO] Replacing original artifact with shaded artifact.
[INFO] Replacing /code/flink/flink-connectors/flink-sql-connector-hive-3.1.3/target/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT.jar with /code/flink/flink-connectors/flink-sql-connector-hive-3.1.3/target/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT-shaded.jar
[INFO] Dependency-reduced POM written at: /code/flink/flink-connectors/flink-sql-connector-hive-3.1.3/target/dependency-reduced-pom.xml
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (integration-tests) @ flink-sql-connector-hive-3.1.3_2.12 ---
[INFO] 
[INFO] --- japicmp-maven-plugin:0.16.0_m325:cmp (default) @ flink-sql-connector-hive-3.1.3_2.12 ---
[INFO] Skipping execution because parameter 'skip' was set to true.
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ flink-sql-connector-hive-3.1.3_2.12 ---
[INFO] Installing /code/flink/flink-connectors/flink-sql-connector-hive-3.1.3/target/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT.jar to /.m2/repository/org/apache/flink/flink-sql-connector-hive-3.1.3_2.12/1.17-SNAPSHOT/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT.jar
[INFO] Installing /code/flink/flink-connectors/flink-sql-connector-hive-3.1.3/target/dependency-reduced-pom.xml to /.m2/repository/org/apache/flink/flink-sql-connector-hive-3.1.3_2.12/1.17-SNAPSHOT/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 26.645 s
[INFO] Finished at: 2022-10-14T01:40:12+05:30
[INFO] ------------------------------------------------------------------------

```